Exercise_1.java

```diff
@@ -6,28 +6,60 @@
     boolean isEmpty() 
     { 
         //Write your code here 
+        if(top== -1){
+            return true;
+        }else{
+            return false;
+        }
+
     } 
 
     Stack() 
     { 
         //Initialize your constructor 
+        this.top = -1;
     } 
   
     boolean push(int x) 
     { 
         //Check for stack Overflow
         //Write your code here
+        if(top>1000){

```

Comment:
1) wrong condition: if (top + 1 == MAX)
2) don't hard code values, use MAX instead of 1000